forms-html/AhmadA.Duhoki-ZainabSaud/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>2nd Day of HTML Test</title>
+</head>
+<body>
+    <form method="POST">
+        <label for="email">Enter Your Email:</label>
+            <br>
+        <input type="email" required name="email" id="email">
+            <br>
+        <label for="username">Username:</label>
+            <br>
+        <input type="text" name="username" id="username">
+            <br>
+        <label for="password">Password:</label>
+            <br>
+        <input type="password" name="password" id="password">
+            <br>
+        <label for="quantity">Quantity(between 1 and 5):</label>
+        <input type="number" max="5" min="1" name="quantity" id="quantity">
+            <br>
```

Comment:
Great job on using the for attribute here. It makes it more convenient for the user as the user can click the label and the text field would get focused.